proprietary/tokens/README.md

```diff
@@ -1,3 +1,229 @@
 <!-- @license CC0-1.0 -->
 
-# TODO: write readme
+# Amsterdam Design System: Tokens
+
+This package provides all design tokens from the [Amsterdam Design System](https://designsystem.amsterdam).
+Use it to apply its visual design to your website or application.
+The tokens are pure CSS, making them compatible with most if not all technology stacks.
```

Comment:
Don't we want to mention some of the other exports we have here? That would make the second part of the sentence a bit more correct. Something like:

```suggestion
The tokens are exported in multiple formats, including CSS custom properties and JSON, making them compatible with most if not all technology stacks.
```